test_local.py

```
@@ -0,0 +1,148 @@
+    # Save original env vars
+    original_env = dict(os.environ)
+    
+    try:
+        # Test missing variables
+        for key in ['investec_client_id', 'cartrack_username', 'rate_per_km']:
+            if key in os.environ:
+                del os.environ[key]
+        
+        try:
+            validate_environment()
+            assert False, "Should have raised ValueError for missing env vars"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)